<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>버튼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 계좌</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_con">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>종목코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>종목이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>현재 가격</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>전일종가</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>전일고가</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="cbox_con">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckbox_real">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>실시간 활성화</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
